### PR TITLE
backport: 'trivial' pr12

### PR DESCRIPTION
**Pull request by PastaPastaPasta**

### CODE REVIEW

File: doc/build-windows.md [hunk 1]
```diff
@@ -9,7 +9,7 @@ The options known to work for building Dash Core on Windows are:
 and is the platform used to build the Dash Core Windows release binaries.
 * On Windows, using [Windows
 Subsystem for Linux (WSL)](https://docs.microsoft.com/windows/wsl/about) and the Mingw-w64 cross compiler tool chain.
-* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com).
+* On Windows, using a native compiler tool chain such as [Visual Studio](https://www.visualstudio.com). See [README.md](/build_msvc/README.md).
```
Comment (PastaPastaPasta) [inline doc/build-windows.md:12]: drop 19061

File: test/functional/wallet_labels.py [hunk 2]
```diff
@@ -190,6 +190,33 @@ def _run_subtest(self, accounts_api, node):
             # already the receiving address of the label. This is a no-op.
             change_label(node, labels[2].receive_address, labels[2], labels[2], accounts_api)
 
+        self.log.info('Check watchonly labels')
```
Comment (PastaPastaPasta) [inline test/functional/wallet_labels.py:193]: drop 18974?

File: src/test/sanity_tests.cpp [hunk 3]
```diff
@@ -14,7 +14,7 @@ BOOST_AUTO_TEST_CASE(basic_sanity)
 {
   BOOST_CHECK_MESSAGE(glibc_sanity_test() == true, "libc sanity test");
   BOOST_CHECK_MESSAGE(glibcxx_sanity_test() == true, "stdlib sanity test");
-  BOOST_CHECK_MESSAGE(ECC_InitSanityCheck() == true, "openssl ECC test");
+  BOOST_CHECK_MESSAGE(ECC_InitSanityCheck() == true, "secp256k1 sanity test");
```
Comment (UdjinM6) [inline src/test/sanity_tests.cpp:17]: 18825:
>OpenSSL is long gone.

not in Dash Core :)

File: configure.ac [hunk 4]
```diff
@@ -373,6 +373,7 @@ if test "x$enable_werror" = "xyes"; then
   if test "x$CXXFLAG_WERROR" = "x"; then
     AC_MSG_ERROR("enable-werror set but -Werror is not usable")
   fi
+  AX_CHECK_COMPILE_FLAG([-Werror=gnu],[ERROR_CXXFLAGS="$ERROR_CXXFLAGS -Werror=gnu"],,[[$CXXFLAG_WERROR]])
```
Comment (UdjinM6) [inline configure.ac:376]: 18887: will probably cause issues for some backports until we backport 18088

File: .fuzzbuzz.yml [hunk 5]
```diff
@@ -0,0 +1,16 @@
+base: ubuntu:16.04
+language: c++
+engine: libFuzzer
+environment:
+  - CXXFLAGS=-fcoverage-mapping -fno-omit-frame-pointer -fprofile-instr-generate -gline-tables-only -O1
+setup:
+  - sudo apt-get update
+  - sudo apt-get install -y autoconf bsdmainutils clang git libboost-all-dev libboost-program-options-dev libc++1 libc++abi1 libc++abi-dev libc++-dev libclang1 libclang-dev libdb5.3++ libevent-dev libllvm-ocaml-dev libomp5 libomp-dev libprotobuf-dev libqt5core5a libqt5dbus5 libqt5gui5 libssl-dev libtool llvm llvm-dev llvm-runtime pkg-config protobuf-compiler qttools5-dev qttools5-dev-tools software-properties-common
+  - ./autogen.sh
+  - CC=clang CXX=clang++ ./configure --enable-fuzz --with-sanitizers=address,fuzzer,undefined
```
Comment (UdjinM6) [inline .fuzzbuzz.yml:10]: 18634: nit: won't work but it's probably ok to merge it as a starting point

File: doc/build-windows.md (hunk 1 above)
Comment (UdjinM6) [reply doc/build-windows.md:12]: Maybe keep it but add a note that it's incomplete?

File: test/functional/wallet_labels.py (hunk 2 above)
Comment (UdjinM6) [reply test/functional/wallet_labels.py:193]: We could probably convert it to test "legacy" addresses instead though it's probably better to do it in a separate PR with a proper commit message. So yeah, let's just skip it for now.

**Review by UdjinM6 — CHANGES_REQUESTED** [review]

File: doc/build-windows.md (hunk 1 above)
Comment (PastaPastaPasta) [reply doc/build-windows.md:12]: What do you mean? We don't have build_msvc yet so we shouldn't do this backport no?

Comment (UdjinM6) [reply doc/build-windows.md:12]: oh, right 🙈 drop it then :)

**Comment by PastaPastaPasta** [comment]
Dropped 18974, 19061, 18887, 18825

**Review by UdjinM6 — APPROVED** [review]
LGTM, utACK

**Pull request merged**